tests/real/enterprise.rs

```diff
@@ -360,4 +359,14 @@ fn ent_unique_job_bypass_unique_lock() {
     } else {
         panic!("Expected protocol error.")
     }
+
+    // let's consume three times from the queue to verify that the first two jobs
+    // have been enqueued for real, while the last one has not.
+    let mut c = ConsumerBuilder::default();
+    c.register("order", |j| -> io::Result<_> { Ok(eprintln!("{:?}", j)) });
+    let mut c = c.connect(Some(&url)).unwrap();
+
+    assert!(c.run_one(0, &[queue_name]).unwrap());
+    assert!(c.run_one(0, &[queue_name]).unwrap());
```

Comment:
to address [comment](https://github.com/jonhoo/faktory-rs/pull/45#discussion_r1443988164)